### ISSUE TITLE
Missing color-ayu* Xresource files

### ISSUE THREAD

**Issue by jsy1001** [regolith-linux/regolith-desktop#189]
I noticed that there are alternative color definitions in /etc/regolith/styles/root that reference missing files /etc/regolith/styles/color-ayu*

These lines are commented out, so nothing is broken unless you try switching to one of these color schemes in a staged copy of the configuration file.

**Comment by kgilmer** [comment]
Hi @jsy1001 , this will be fixed in the upcoming 1.3 release.

**Comment by kgilmer** [comment]
The color files are shipped now but the packages express dependencies that are unavailable in the PPA and must be installed manually like this:
```
 wget https://github.com/appelgriebsch/ayu-theme/releases/download/ayu-0.1.0/ayu-theme_0.1.0-1_amd64.deb

 wget https://github.com/appelgriebsch/ayu-theme/releases/download/ayu-mirage-0.1.0/arc-icon-theme_20161122-1_amd64.deb

sudo dpkg -i arc-icon-theme_20161122-1_amd64.deb

sudo dpkg -i ayu-theme_0.1.0-1_amd64.deb
```
